github-actions/feature-request/src/main.ts

```diff
@@ -0,0 +1,48 @@
+import * as core from '@actions/core';
+import { context } from '@actions/github';
+import { run } from './action';
+import { OctoKit } from './octokit';
+
+// Gets a specific value from the YAML configuration.
+// The value could be either a number or a string.
+const v = <T extends number|string|boolean>(name: string): T => {
```

Comment:
I think it would be better to have this be named something more intuitive.

Maybe `getInputValue`?